haiku-apps/conway/conway-0.9.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A simple implementation of Conway's \"Game of Life\""
+DESCRIPTION="Conway is a fascinating artificial-life simulation. \
+It differs from the \"Life\" program included with BeOS in 2 ways: \
+first, it runs in a window, not full-screen; and second, it provides \
+a delta display showing how the board has changed over time."
+HOMEPAGE="https://github.com/HaikuArchives/Conway"
+COPYRIGHT="1998 Alex Chamberlain"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="cc589fdda35b4f5b07c1bd3a4dc797638a45b50e"
+SOURCE_URI="https://github.com/HaikuArchives/Conway/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="18e489cd650a862e937606ad9ecac4b840ee65dd005cf780c844c258fd2d4bbd"
+SOURCE_FILENAME="Conway-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Conway-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	conway = $portVersion
+	app:Conway = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:make
+	"
+
+defineDebugInfoPackage conway \
+ 	$appsDir/Conway
```

Comment:
Oops, there's a space before the tab (in line 36).

Reply:
Do you think we should rename the recipe to `conway-0.9~git.recipe` ?
(According to our rules, it would be better, but since the app is probably not going to change, I guess we can keep the current name.)

Reply:
Sorry, I just noticed that the space came from the diff I wrote in my comment above.
(I got rid of it, and then merged the PR, so we're done with this recipe :)